examples/zkapps/06-offchain-storage/experimental-zkapp-offchain-storage/README.md

```diff
@@ -0,0 +1,113 @@
+# Mina zkApp: Zkapp Offchain Storage
+
+An experimental library for storing zkApp data offchain. This project is currently under development and should not be used in production.
+
+## Usage
+
+### Dependencies:
+
+Ensure all required modules are installed. To do so, run the following command:
+
+```bash
+npm install
+```
+
+### Running the Example
+
+#### Blockchain Mode
+
+Toggle the `useLocalBlockchain`` variable to switch between local testing and real-world usage.
+
+#### Account Configuration:
+
+For local testing, predefined accounts will be used.
+For `Berkeley` mode, you must provide a `deployerPrivateKey` and `zkAppPrivateKey` as command line arguments.
+
+To generate these keys, you can use the following command:
+
+```bash
+npm run generate-keys
+
+```
+
+### Running the Application:
+
+Ensure that the offchain server is running. To do so, run the following command:
+
+```bash
+npm run server
```

Comment:
It looks like this script is broken.  I received this error. 
```
cannot find module 'docs2/tutorial-6/docs2/examples/zkapps/06-offchain-storage/experimental-zkapp-offchain-storage/build/src/storageServer.js'
```

The current package.json script is trying to access the wrong file path.

```
"server": "npm run build && node --enable-source-maps build/src/storageServer.js"
```
 This worked for me. I will add a comment to the package.json file also.
 
     "server": "npm run build && node --enable-source-maps build/src/server/storageServer.js"